backend/oasst_backend/scheduled_tasks.py

```diff
@@ -0,0 +1,133 @@
+from __future__ import absolute_import, unicode_literals
+
+import os
+from datetime import datetime
+from typing import Any, Dict, List
+
+from asgiref.sync import async_to_sync
+from celery import Celery, shared_task
+from dotenv import load_dotenv
+from loguru import logger
+from oasst_backend.models import ApiClient
+from oasst_backend.prompt_repository import PromptRepository
+from oasst_backend.user_repository import User
+from oasst_backend.utils.database_utils import default_session_factory
+from oasst_backend.utils.hugging_face import HfClassificationModel, HfEmbeddingModel, HfUrl, HuggingFaceAPI
+from oasst_shared.utils import utcnow
+from sqlmodel import select
+
+result = load_dotenv(".env")
```

Comment:
This code in global scope looks not good. why is it here? Is this file an entry-point?

Reply:
Yes this is the entry function for the Celery Worker process